protocol/swap/manager.go

```diff
@@ -148,13 +148,15 @@ func (m *manager) GetOngoingSwap(id types.Hash) (Info, error) {
 }
 
 // GetOngoingSwaps returns all ongoing swaps.
-func (m *manager) GetOngoingSwaps() ([]Info, error) {
+func (m *manager) GetOngoingSwaps() ([]*Info, error) {
```

Comment:
It's kind of a big data structure to be passing around by value. Some places were passing it by value, others by reference. All the places are passing it by reference now.